.gitignore

```diff
@@ -1,4 +1,5 @@
+lastIngest.json
```

Comment:
Does this mean that the `lastIngest.json` is written at the root? Not sure that's the best place for it - can it be moved to the API folder?